### DISCUSSION TITLE
[Bug]: Square Template Link to download not working

### DISCUSSION THREAD

**Discussion by raramuridesign** [GetPublii/Publii#1745] [General]
### Operating system

n/a

### Publii version

n/a

### Post editor

None

### Bug description

Go here, the button sais Purchase not Download, so I guess this needs to be updated.
https://marketplace.getpublii.com/themes/square/

M

**Comment by bobmitro** [comment]
Fixed, thanks for reporting.

**Reply by raramuridesign** [reply]
No problem....
